test/integration/model.test.js

```diff
@@ -2664,7 +2664,40 @@ describe(Support.getTestDialectTeaser('Model'), function() {
     return this.sequelize.sync({force: true});
   });
 
-  describe('bulkCreate errors', function() {
+  describe('bulkCreate', function() {
```

Comment:
OK.
Where is the best place?
Is it OK to create another `describe` at the same level as `bulkCreate errors` in test/integration/model.test.js?